docs/topic-guides/blendable-web-apps/common-look-and-feel.md

```diff
@@ -0,0 +1,84 @@
+# Common look and feel of apps
+
+## Introduction
+
+Good user experience is significant for a successful solution of apps. An author of a single app aims to balance a good default look that matches other apps and to have distinctive features that are both easy to use and pleasing to the eye of the end user.
+
+Starcounter offers a layered approach that allows app authors to adapt to a common look and feel for the typical parts of the UI and to provide their own components for the unique parts. Solution owners can theme or replace the look and feel without changing of the app source code.
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a standards-based layered approach that separates the responsibilities of the app author, the design system, and the solution owner. 
+
+The full design consists of:
+
+1. Content defined by native HTML
+2. Common UI patterns and components of Uniform design system
+3. Theme configuration through custom CSS properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Content defined by native HTML
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the native semantic HTML elements that can be rendered on the screen directly or with their appearance changed in the presentation layer.
+
+### Common UI patterns and components of Uniform design system
+
+As the basic way for providing a common look and feel in the presentation layer, Starcounter provides Uniform design system. It consists of custom CSS properties, CSS reset and a library of patterns, and components.
+
+Uniform follows the inversion of control principle, which makes it a link between app authors and solution owners.
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+The custom CSS properties are the basic set of primitives. They define the fonts, sizes, colors, margins and other values used throughout the reset, patterns and components.
```

Comment:
> The custom CSS properties are the basic set of primitives.

"basic set of primitives" is redundant. Primitives are always basic. 

I would phrase it something like this instead:

> The basic building blocks for the Uniform design system are custom CSS properties. 

Not sure if we need the word "custom" in there; aren't all CSS properties customizable?

Reply:
> Not sure if we need the word "custom" in there; aren't all CSS properties customizable?

Disregard they. they seem to be called custom CSS properties everywhere, so we should use it as well.

Reply:
"Basic building blocks" sound a little bit too cliche to me. Also, I would like Uniform components to be seen as building blocks. 

I will change it to "primitives". These things are sometimes called "design tokens", but we already use the word "token" for something else.

I will just change "the basic set of primitives" to "the set of primitives"